packages/sit-onyx/src/components/OnyxTable/OnyxTable.stories.ts

```diff
@@ -112,3 +111,21 @@ export const LimitedHeightAndWidth = {
     style: "max-width: 20rem; max-height: 16rem",
   },
 } satisfies Story;
+
+/**
+ * This example shows a table without a `tbody`.
+ */
+export const Empty = {
+  args: {
+    default: () => [getTableHeader()],
+  },
+} satisfies Story;
+
+/**
+ * This example shows a table with an empty `tbody` and without a `thead`.
+ */
+export const EmptyWithoutHeader = {
```

Comment:
Not sure if we need to create an example for this since usually the most tables will have headers :)

Reply:
ok :) it's still covered by playwright